pygmt/__init__.py

```diff
@@ -12,16 +12,17 @@
 from pkg_resources import get_distribution
 
 # Import modules to make the high-level GMT Python API
-from .session_management import begin as _begin, end as _end
+from . import datasets
```

Comment:
The `from .session_management import begin as _begin, end as _end` needs to come before the other stuff I think, is there a way to 'allowlist' this so it doesn't get sorted to the bottom?

Reply:
It's unnecessary, but `from . import datasets` must be imported at the end. Otherwise, we will see the below errors:

```python
In [1]: import pygmt
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-f58669fae37f> in <module>
----> 1 import pygmt

~/Gits/gmt/pygmt/pygmt/__init__.py in <module>
     14 # Import modules to make the high-level GMT Python API
     15 from .session_management import begin as _begin, end as _end
---> 16 from . import datasets
     17 from .figure import Figure
     18 from .filtering import blockmedian

~/Gits/gmt/pygmt/pygmt/datasets/__init__.py in <module>
      3 # Load sample data included with GMT (downloaded from the GMT cache server).
      4
----> 5 from .earth_relief import load_earth_relief
      6 from .tutorial import (
      7     load_japan_quakes,

~/Gits/gmt/pygmt/pygmt/datasets/earth_relief.py in <module>
      5 import xarray as xr
      6
----> 7 from .. import grdcut, which
      8 from ..exceptions import GMTInvalidInput
      9 from ..helpers import kwargs_to_strings

ImportError: cannot import name 'grdcut' from partially initialized module 'pygmt' (most likely due to a circular import) (/Users/seisman/Gits/gmt/pygmt/pygmt/__init__.py)
```

Reply:
The circular import issue is fixed in commit 11267db38a5771a4db9937593928a91b7b19c3da.